building.sk

```diff
@@ -215,7 +215,7 @@ old blocks before flattening:
 
 	# Miscellaneous
 	[unlit] pumpkin¦s = minecraft:pumpkin
-	(lit pumpkin|jack-o[']-lantern)¦s = minecraft:lit_pumpkin
+	(lit pumpkin|jack[(-| )]o['][(-| )]lantern)¦s = minecraft:lit_pumpkin
```

Comment:
These dashes/spaces shouldn't be optional, `jackolantern` isn't a valid way to refer to it. Agreed on spaces being good alternatives, though, so just make these into `(-| )`.

Reply:
Will do. I will make that change